aspnetcore/blazor/debug.md

```diff
@@ -1,123 +1,180 @@
+* Resume code execution with <kbd>F8</kbd> in a browser or <kbd>F5</kbd> in Visual Studio or Visual Studio Code.
+* In the *Locals* display, observe the values of local variables.
 * See the call stack, including call chains that go from JavaScript into .NET and from .NET to JavaScript.
 
 You *can't*:
 
-* Observe the values of any locals that aren't an `int`, `string`, or `bool`.
-* Observe the values of any class properties or fields.
-* Hover over variables to see their values.
-* Evaluate expressions in the console.
-* Step across async calls.
-* Perform most other ordinary debugging scenarios.
+* Inspect arrays.
+* Hover to inspect members.
+* Step debug into or out of managed code.
+* Have full support for inspecting value types.
+* Break on unhandled exceptions.
+* Hit breakpoints during app startup.
+* Debug an app with a service worker.
 
-Development of further debugging scenarios is an on-going focus of the engineering team.
+We expect to continue to improve the debugging experience in future releases.
 
 ## Prerequisites
 
 Debugging requires either of the following browsers:
 
+* Microsoft Edge (version 80 or later)
 * Google Chrome (version 70 or later)
-* Microsoft Edge preview ([Edge Dev Channel](https://www.microsoftedgeinsider.com))
+
+## Enable debugging for Visual Studio and Visual Studio Code
+
+To enable debugging in an existing Blazor WebAssembly app in Visual Studio or Visual Studio Code, update *launchSettings.json* in the startup project of your app to include the following `inspectUri` property in each launch profile:
+
+```json
+"inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}"
+```
+
+Once updated, the *launchSettings.json* file should look something like the following:
+
+```json
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:50454",
+      "sslPort": 44399
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "BlazorApp1.Server": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "applicationUrl": "https://localhost:5001;http://localhost:5000",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
+  }
+}
+```
+
+The `inspectUri` property:
+
+* Enables the IDE to detect that the app is a Blazor WebAssembly app.
+* Instructs the script debugging infrastructure to connect to the browser through Blazor's debugging proxy.
 
 ## Procedure
```

Comment:
Instead of having "Procedure" as section that shows up in the right nav, can we remove the Procedure heading and elevate the following sections to be h2s?